handlers.go

```diff
@@ -22,11 +22,11 @@ var (
 	surfacePages = map[string]struct{}{"signup": {}, "login": {}}
 )
 
-func servePublicFile(w http.ResponseWriter, r *http.Request) {
+func getPublic(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could you explain why you decided to change the name for this function?

Reply:
I decided to make all the function names for handlers very clear. get[RequestName] for GET requests, and post[RequestName] for POST requests

Reply:
Okay, if this is the format from now on then please update documentation to reflect this as a standard for the code base.

Reply:
Where would you put this? Its a file full of url request handlers, so its kind of in context that the requests can be of different kinds.